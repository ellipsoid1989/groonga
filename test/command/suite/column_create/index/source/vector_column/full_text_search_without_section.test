plugin_register functions/index_column

table_create Docs TABLE_NO_KEY
column_create Docs sentences COLUMN_VECTOR Text

table_create Words TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
column_create Words docs_sentences COLUMN_INDEX|WITH_POSITION Docs sentences

dump
table_remove Words
