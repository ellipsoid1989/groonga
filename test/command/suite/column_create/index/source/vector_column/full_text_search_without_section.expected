plugin_register functions/index_column
[[0,0.0,0.0],true]
table_create Docs TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Docs sentences COLUMN_VECTOR Text
[[0,0.0,0.0],true]
table_create Words TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
[[0,0.0,0.0],true]
column_create Words docs_sentences COLUMN_INDEX|WITH_POSITION Docs sentences
[
  [
    [
      -22,
      0.0,
      0.0
    ],
    "grn_obj_set_info(): GRN_INFO_SOURCE: full text index for vector column must be created with WITH_SECTION flag: <Words.docs_sent"
  ],
  false
]
#|e| grn_obj_set_info(): GRN_INFO_SOURCE: full text index for vector column must be created with WITH_SECTION flag: <Words.docs_sentences>
dump
plugin_register functions/index_column

table_create Docs TABLE_NO_KEY
column_create Docs sentences COLUMN_VECTOR Text

table_create Words TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
table_remove Words
[[0,0.0,0.0],true]
